ddmrp_report_part_flow_index/views/ddmrp_flow_index_group_views.xml

```diff
@@ -0,0 +1,70 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2024 ForgeFlow S.L. (https://www.forgeflow.com)
+     License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl-3.0) -->
+<odoo>
+
+    <record id="view_ddmrp_flow_index_group_tree" model="ir.ui.view">
+        <field name="name">ddmrp.flow.index.group.tree</field>
+        <field name="model">ddmrp.flow.index.group</field>
+        <field name="arch" type="xml">
+            <tree>
+                <field name="name" />
+            </tree>
+        </field>
+    </record>
+
+    <record id="view_ddmrp_flow_index_group_form" model="ir.ui.view">
+        <field name="name">ddmrp.flow.index.group.form</field>
+        <field name="model">ddmrp.flow.index.group</field>
+        <field name="arch" type="xml">
+            <form>
+                <sheet>
+                    <div class="oe_button_box" name="button_box">
+                        <button
+                            name="toggle_active"
+                            type="object"
+                            class="oe_stat_button"
+                            icon="fa-check"
+                        >
+                            <field
+                                name="active"
+                                widget="boolean_button"
+                                options='{"terminology": "active"}'
+                            />
+                        </button>
```

Comment:
This needs migration

![image](https://github.com/OCA/ddmrp/assets/23449160/7ba55440-1983-435f-8404-9c120734470e)